src/Controller/FeedController.php

```diff
@@ -45,7 +45,8 @@ public function heyLoyalty(): JsonResponse
 
         $categories = $this->entityManager->getRepository(Category::class)->findByMaterialDate($date);
 
-        $sortKey = 0;
+        // heyloyalty does text sort, so we need to 'pad' the sort key
+        $sortKey = 100000;
```

Comment:
Maybe a little more information about why "text sort" is fixed by starting at the magic value `10000`

Reply:
Updated comment